### ISSUE TITLE
Add section regarding keyboard shortcuts

### ISSUE THREAD

**Issue by larshp** [abapGit/docs.abapgit.org#59]
there is https://docs.abapgit.org/guide-ui.html

Suggest renaming "UI features" -> "Keyboard Navigation", add the missing sections

and move the todo's to new issues(assuming they do not already exist as issues)
